src/components/FilterOptions.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import * as Types from "../utils/Types";
+import {
+  handleInput,
+  handleVisibilityToggle,
+  handleSearchBarToggle,
+} from "../utils/EventHandlers";
+
+const FilterOptions = (props: Types.FilterOptionsProps) => {
+  const [state] = props.stateManager;
+  return (
+    <div className="filter-options">
+      <input
+        className="searchbar"
+        placeholder="Enter institution, lab, or address"
+        onChange={handleInput(props.stateManager)}
+      />
+      <button onClick={handleVisibilityToggle(props.stateManager)}>
+        {`Toggle table row filter by visibility: ${state.useMarkerVisibility.toString()}`}
+      </button>
+      <button onClick={handleSearchBarToggle(props.stateManager)}>
```

Comment:
I don't want this to be toggleable. You should add a clear button to the search bar.